api/pom.xml

```diff
@@ -104,7 +104,7 @@
 		<dependency>
 			<groupId>org.bahmni.module</groupId>
 			<artifactId>teleconsultation-api</artifactId>
-			<version>1.0.0-SNAPSHOT</version>
+			<version>${teleconsultation.version}</version>
```

Comment:
Also, it shouldn't be necessary to specify the version / type or scope for this dependency since those are all defined in the parent `dependencyManagement` (which is inherited).